e2e/wdio/serialization/nextjs/next.module.json

```diff
@@ -0,0 +1,13 @@
+{
+  "namespace": "AllContent",
+  "items": {
+    "includes": [
+      {
+        "allowedPushOperations": "createUpdateAndDelete",
+        "name": "content",
+        "path": "/sitecore/content",
```

Comment:
This file looks specific to the next js sample app, but here we're targeting the entire content tree. Should this be more specific e.g. `/sitecore/content/JssNextWeb`?